tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -1873,6 +1873,32 @@ static void pa_sync_synced_cb(struct bt_le_per_adv_sync *sync,
 {
 	LOG_DBG("");
 
+#if defined(CONFIG_BT_PER_ADV_SYNC_TRANSFER_RECEIVER)
+	/* sync transfer received */
+	if (info->conn) {
+		struct btp_gap_ev_periodic_transfer_received_ev ev;
+		struct bt_conn_info conn_info;
+
+		if (pa_sync) {
```

Comment:
```suggestion
		if (pa_sync != NULL) {
```

Should we log or fail if we already have a PA sync here?